File: 1395-minimum-time-visiting-all-points/minimum-time-visiting-all-points.cpp
class Solution {
public:
    int minTimeToVisitAllPoints(vector<vector<int>>& p) {
        int ans=0;
        for(int i=1;i<p.size();i++){
            int x=abs(p[i][0]-p[i-1][0]);
            int y=abs(p[i][1]-p[i-1][1]);
            ans+=max(x,y);
        }
        return ans;
    }
};

File: 1395-minimum-time-visiting-all-points/minimum-time-visiting-all-points.java
class Solution {
    public int minTimeToVisitAllPoints(int[][] p) {
           int ans=0;
        for(int i=1;i<p.length;i++){
            int x=Math.abs(p[i][0]-p[i-1][0]);
            int y=Math.abs(p[i][1]-p[i-1][1]);
            ans+=Math.max(x,y);
        }
        return ans;
    }
}